_episodes/08-connected-components.md

```diff
@@ -11,38 +11,48 @@ objectives:
 - "Use CCA to produce an image that highlights every object in a different color."
 - "Characterize each object with numbers that describe its appearance."
 keypoints:
-- "`skimage.measure.label` is used to generate objects."
-- "We use `skimage.measure.regionprops` to measure properties of labelled objects."
-- "Color objects according to feature values."
+- "We can use `skimage.measure.label` to find and label connected objects in an image."
+- "We can use `skimage.measure.regionprops` to measure properties of labeled objects."
+- "We can display the labeled image to view the objects colored by label."
 ---
 
-
 ## Objects
 
-In the [thresholding episode]({{ page.root }}/07-thresholding) we have covered dividing an image in foreground and background pixels.
-In the junk example image, we considered the colored shapes as foreground _objects_ on a white background.
+In the [thresholding episode]({{ page.root }}/07-thresholding) we have
+covered dividing an image in foreground and background pixels.  In the
```

Comment:
```suggestion
covered dividing an image into foreground and background pixels.  In the
```

Reply:
Addressed in a82d659bb036937700f773606fb2e7b03f2e7a3a.